go/chat/storage/storage.go

```diff
@@ -569,7 +569,7 @@ func (s *Storage) handleDeleteHistory(ctx context.Context, convID chat1.Conversa
 // Returns whether local deletes happened.
```

Comment:
Update comment to something like "Returns an Expunge only if local messages were deleted"